src/main/java/cz/it4i/parallel/ui/ParadigmScreenController.java

```diff
@@ -204,6 +209,28 @@ private void initParadigms() {
 			.getPlugins())
 		{
 			paradigms.getItems().add(info.getPluginClass());
+			
+			paradigms.setConverter(new StringConverter<Class<? extends ParallelizationParadigm>>() {
+				@Override
+				public String toString(Class<? extends ParallelizationParadigm> item) {
+					if (item == null) {
+						return "";
+					} else {
+						return "" + item.getSimpleName();
+					}
+				}
+
+				@Override
+				public Class<? extends ParallelizationParadigm> fromString(String s) {
```

Comment:
Consider leave empty body o "fromString" with throwing UnsupportedOperationException - this method is not intended to use.